Runtime/Scripts/PeriodicSnapshots.cs

```diff
@@ -16,6 +16,9 @@ public class PeriodicSnapshots : MonoBehaviour
         [DllImport("__Internal")]
         private static extern void DownloadFile(string gameObjectName, string methodName, string filename, byte[] byteArray, int byteArraySize);
 
+        [DllImport("__Internal")]
```

Comment:
Graag 

```
[DllImport("__Internal")]
        private static extern void DownloadFromIndexedDB(string filename, string callbackObjectName, string callbackMethodName);
```

gebruiken ipv DownloadFile en TriggerDownload. Helaas is dit een artefact van een oude oplossing die we hadden, deze methode wordt nu maar matig ondersteunt en zit nog in de FileBrowser vanwege BC.

Je kan de `ProjectDataStore` class bekijken voor een implementatie